content/_includes/header.html

```diff
@@ -1,22 +1,14 @@
 <header class="site-header">
   <div class="container">
-    <nav class="navbar navbar-expand-lg navbar-light bg-light top-nav">
-      <div class="nav-wrap">
-        <a class="navbar-brand site-logo-link" href="{{ "/" | relative_url }}"><img class="site-logo-img" src="{{ "/CWL-Logo-Header.png" | relative_url }}" alt="Common Workflow Language logo" height="61" width="106"></a>
-        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#topNav" aria-controls="topNav" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
-        </button>
-      </div>
-
-      <div class="collapse navbar-collapse" id="topNav">
-        {% include top_nav.html %}
-        <!-- <form class="form-inline my-2 my-lg-0 search-bar-top">
-          <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
-          <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
-        </form> -->
-      </div>
-    </nav>
-
+    <div class="site-header-logo">
+      <h1 class="sr-only">Common Workflow Language</h1>
+      <a class="" href="{{ site.url }}"><img class="" src="{{ "assets/img/CWL-Logo-HD-cropped2.png" | relative_url }}" alt="Common Workflow Language logo"></a>
+    </div>
   </div>
-
 </header>
+
+{% if page.class contains "has-announcement" %}
+  <div class="cwl-announcement"><h3> 2022 CWL Conference (Feb 28 - Mar 4, 2022)</h3></div>
```

Comment:
```suggestion
  <div class="cwl-announcement"><h3><a href="https://cwl.discourse.group/t/2022-cwl-conference-feb-28-mar-4-2022/519">2022 CWL Conference</a> (Feb 28 - Mar 4, 2022)</h3></div>
```

Reply:
I think we don't want an blue url here - and are going to make the banner a clickable link instead.